build_images.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# required for `skipper` according to
+# https://github.com/Stratoscale/skipper/blob/upstream/README.md#python3-environment
+export LC_ALL="en_US.UTF-8"
+export LANG="en_US.UTF-8"
+
+TAG=$(git rev-parse --short=7 HEAD)
+ASSISTED_SERVICE_IMAGE=quay.io/app-sre/assisted-service
+ASSISTED_ISO_GENERATOR_IMAGE=quay.io/app-sre/assisted-iso-generator
```

Comment:
isn't the image called `assisted-iso-create`?

Reply:
Not on `app-are`, it will be changed lated to `assisted-iso-create` https://issues.redhat.com/browse/MGMT-1930
@filanov